activities/services/timeline.py

```diff
@@ -82,3 +82,16 @@ def identity_public(self, identity: Identity):
             .unlisted(include_replies=True)
             .order_by("-id")
         )
+
+    def favourites(self) -> models.QuerySet[Post]:
```

Comment:
Our convention is to call these `likes` everywhere except where we interface with Mastodon - so here and the tests should use like, and the mastodon API is the thing that's called favourites.

Reply:
Alright, all updated :slightly_smiling_face: